CMakeLists.txt

```diff
@@ -48,11 +48,14 @@ if(NOT DEFINED ONNX_VERIFY_PROTO3)
   option(ONNX_VERIFY_PROTO3 "Generate code by proto3" ${PROTO3_ENABLED})
 endif()
 
- if (ONNX_USE_PROTOBUF_SHARED_LIBS)
+if(ONNX_USE_PROTOBUF_SHARED_LIBS)
```

Comment:
The comment for the option should also be updated to reflect that the flags for using static libs will be impaced:

```
option(ONNX_USE_PROTOBUF_SHARED_LIBS "Build ONNX using protobuf shared library. Sets PROTOBUF_USE_DLLS CMAKE Flag " OFF)
```